CybridSDK/View Layer/IdentityComponent/View/IdentityVerificationViewModel.swift

```diff
@@ -250,9 +252,28 @@ class IdentityVerificationViewModel: NSObject {
 
             UIState.value = .REVIEWING
 
+        case .expired:
+
+            getIdentityVerificationStatus(identityWrapper: nil)
+
         default:
 
             UIState.value = .ERROR
         }
     }
 }
+
+class IdentityVerificationWrapper: Equatable {
+
+    static func == (lhs: IdentityVerificationWrapper, rhs: IdentityVerificationWrapper) -> Bool {
+        return true
```

Comment:
Always return `true` here? Shouldn't we look at the `guid` or something similar?